engine/src/main/java/org/terasology/entitySystem/sectors/SectorSimulationComponent.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.entitySystem.sectors;
+
+import org.terasology.entitySystem.Component;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.internal.BaseEntityRef;
+import org.terasology.module.sandbox.API;
+import org.terasology.protobuf.EntityData;
+
+/**
+ * The component that allows the {@link SectorSimulationSystem} to send simulation events to a sector-scope entity.
+ *
+ * This should be automatically added by either {@link EntityManager#createSectorEntity(long)} or
```

Comment:
I was thinking about that too. There should probably be a way to define this in the prefab, and to set the maxDelta there too.

I think it would be useful, as you can just create an entity from the prefab, and not worry about any scope or deltas or anything like that (which would also allow, say, a `DeerGroup` to simulate correctly if it is spawned with the `spawnPrefab` command).